cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt

```diff
@@ -375,6 +375,7 @@ Would have sent complete blob: {
         "unsupported_exts": {
             ".txt": 1
         },
+        "lockfile_scan_info": {},
```

Comment:
I don't think there's any test that snapshots this as not empty

Reply:
You're right. Will add one

Reply:
Well now they all have non-empty dicts here, which should be ok I think.